### PR TITLE
Build Python 3.7 Wheels

### LINKED ISSUE #2: Build wheels for Python 3.7
Some multibuild adjustments might be needed.

### PR DESCRIPTION
**Pull request by jeremiahpslewis**
Based on Matplotlib 3.7 configuration, I believe this could do the trick for Python 3.7. Would very much appreciate feedback here, happy to dig further.

E.g. trying to resolve #2 @riddell-stan.
https://github.com/MacPython/matplotlib-wheels/blob/master/.travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -78,7 +87,12 @@ matrix:
       env:
         - MB_PYTHON_VERSION=3.6
         - NP_BUILD_DEP="1.9.3"
-
+    - os: osx
+      language: generic
+      env:
+        - MB_PYTHON_VERSION=3.7
+        - NP_BUILD_DEP=1.14.5
+        - NP_TEST_DEP=1.14.5
```
Comment (riddell-stan) [inline .travis.yml]: this repo doesn't use ``NP_TEST_DEP`` -- maybe remove this?

**Comment by riddell-stan** [comment]
Looks promising. Does this work for you locally?

I think the multibuild repo might need to be updated to solve the other failures.

File: .travis.yml (hunk 1 above)
Comment (jeremiahpslewis) [reply .travis.yml]: Thanks. Fixed.

**Comment by jeremiahpslewis** [comment]
Looks like the build passes now. Would love a review & merge as appropriate. @riddell-stan

**Comment by riddell-stan** [comment]
Thanks for figuring out what was wrong. It's a great help.

**Comment by jeremiahpslewis** [comment]
Thank you for maintaining this library, it's a far bigger contribution. 🥂 🍻

**Comment by riddell-stan** [comment]
Wheels now on PyPI.

**Pull request merged**